vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/RemoveSortableColumnPage.java

```diff
@@ -140,7 +140,6 @@ public RemoveSortableColumnPage() {
         grid.setHeaderGenerator(getHeaderGenerator());
 
         add(grid);
-        grid.redraw();
 
         Button sort = new Button("Sort");
         sort.setId("sort");
```

Comment:
I suggest you also change this id to something different. 
This is a test I did in a random test view.

![Screenshot from 2021-05-05 19-58-03](https://user-images.githubusercontent.com/28783969/117180024-6a22ad80-addc-11eb-96b8-14f6e041f4c4.png)

Reply:
Thanks Tulio that actually was the issue 👍. Strangely that only showed up when running the combined tests. Anyway it's fixed now.